brkp.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "map.h"
+#include "riscv.h"
+
+typedef struct {
+    riscv_word_t addr;
+    uint32_t orig_instr;
+} brkp_t;
+
+typedef struct {
+    map_t map;
+} brkp_map_t;
```

Comment:
Why not declaring `typedef map_t brkp_map_t;` straightforwardly?

Reply:
I am not sure if we will extend this structure. I may want to record something like the reference to the previous hit breakpoint.